Jenkinsfile

```diff
@@ -26,11 +26,8 @@ pipeline {
             when { branch 'main' }         // Runs only when it's a PR .dd
             steps {
                 sh "env"
-                sh "echo I am Main Branch"           
+                sh "echo I am Running Against The Main Branch"           
```

Comment:
Looks good to me.
Can you add the job run against the PR.

Reply:
@trainer-manuverma  Here is the job ref of the job againt the PR.
http://18.233.111.83:8080/job/ansible-promote-mb/view/change-requests/job/PR-4/